### PR TITLE
Fixed typos.

### PR DESCRIPTION
**Pull request by amitmerchant1990**
Hi Team,

I was going through the tutorial on Tour of Heroes and found out these typos in the documentation. Hope these will make the documentation better.

Thanks!

### CODE REVIEW

**Comment by amitmerchant1990** [comment]
I signed the Google CLA!

**Comment by Foxandxss** [comment]
The `will` goes against our guidelines, so we cannot accept this PR.

**Comment by amitmerchant1990** [comment]
Which guideline rule is this violating?

**Comment by Foxandxss** [comment]
Our internal guideline in how to write documentation. We avoid the use of `will`.

**Comment by amitmerchant1990** [comment]
@Foxandxss Okay. Got it!

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 1]
```diff
@@ -239,7 +239,7 @@ block router-config-intro
 
     We've setup initial routes in the `app.routing.ts` file. Now we'll add it to our root NgModule.
 
-    Import the `routing` constant from `app.routing.ts` and add it the `imports` !{_array} of `AppModule`.
+    Import the `routing` constant from `app.routing.ts` and add it into the `imports` !{_array} of `AppModule`.
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: This is a true type. Thanks.

**Comment by wardbell** [comment]
Closing because we do not agree that these are typos ... except the one

**Pull request closed without merging**